CHANGELOG.md

```diff
@@ -64,6 +69,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   literals. This means that `snapshot.max-new-file-size="1"` and
   `snapshot.max-new-file-size=1` are now equivalent.
 
+* ED25519 host keys are now supported correctly when connecting to Git+SSH hosts
```

Comment:
I'm hopeful that this will actually fix several other git ssh issues, but I guess we shouldn't get ahead of ourselves :)